### DISCUSSION TITLE
Resending for pattern problem

### DISCUSSION THREAD

**Discussion by medusasaur0** [bitcraze/discussions#736] [Q&A]
I wanted to connect to my old crazyflie 1.0 but encountered this problem.
![image](https://github.com/bitcraze/discussions/assets/104261276/4ee220c3-8721-4e70-9cda-1e4bc376819e)
I tried latest versions, and old versions of cflib and virtual machines from 2017, where there still was support for cf 1.0. But on every version cfclient is resending pattern (76, 1). How can i fix this?

**Comment by medusasaur0** [comment]
I've updated firmware on crazyradio, but still resending for pattern(76, 1). I think it's crazyflie firmware problem, because when i try to bootload crazyflie it gives me every time variouse errors, anyone knows how to fix this?

**Comment by medusasaur0** [comment]
trying to bootload gives me this error
![image](https://github.com/bitcraze/discussions/assets/104261276/60abb77d-38ee-401d-aaa1-befdbc6141af)

**Comment by ataffanel** [comment]
I am not sure what is happening exactly but my guess is that the versions of lib/client/crazyflie firmware do not match. I would suggest to make sure all the version matches (ie. they where released about at the same date).

The error you are getting in the boot-loader is unfortunate. It does feel like something that would happen when the lib and the client version are too far apart release-date-wise.

**Reply by medusasaur0** [reply]
I was using cfclient 2017.3, 2017.6, 2017.6.1 and cflib 0.1.1, 0.1.3. They all match, I'll try older versions
